dev/RadioButtons/RadioButtons.h

```diff
@@ -74,6 +74,7 @@ class RadioButtons :
 
     int m_selectedIndex{ -1 };
     bool m_currentlySelecting{ false };
+    bool m_blockSelecting{ true };
```

Comment:
What is the difference in behavior / invariant for m_currentlySelecting versus m_blockSelecting? Some comments here would help.

Reply:
currentlySelecting is the property to block against reentrency, since calling select changes the SelectedIndex and Item which in turn calls select.  blockSelecting is used to.. block selecting :). In this case I'm using it to not change selection before we have loaded the radio buttons, this prevents things like the creation of the Items vector from chaning selection which was trampeling the selected index property set by the consumer.